x-pack/plugins/security_solution/public/management/components/hooks/use_url_pagination.ts

```diff
@@ -64,6 +64,20 @@ export const useUrlPagination = (): UrlPagination => {
   const location = useLocation();
   const history = useHistory();
   const { urlParams, toUrlParams } = useUrlParams();
+
+  useEffect(() => {
```

Comment:
@paul-tavares let me know your thoughts on this. I decided to put this at this level as can be useful when using this hook in other places when migration the list pages

Reply:
@dasansol92 - Yeah, having it here would address the problem, but it feels really wrong (to me 😄 ) that we are doing a `history.replace()` in a hook meant to "get" url params - especially when it is being done automatically. 

Also, the migration below is from `page_index` to `page`, which may not be an accurate mapping - `page_index` is usually `0` based, where we now want to start using `page` as `1` based in order to provide better URL UX to the users. _(ex. User clicks on page `2`, they see `2` in the url)_

Reply:
That's totally right. Then we can create a specific hook for it and use it in the needed components or move this code directly to the specific component. Thoughts?